rrdd/rrdd_http_handler.ml

```diff
@@ -67,23 +67,23 @@ let get_host_stats ?(json = false) ~(start : int64) ~(interval : int64)
     ~(sr_uuid : string) () =
   Mutex.execute mutex (fun () ->
       let prefixandrrds =
-        let vm_rrds = Hashtblext.to_list vm_rrds in
-        let sr_rrds = Hashtblext.to_list sr_rrds in
+        let vm_rrds = Hashtbl.to_seq vm_rrds in
+        let sr_rrds = Hashtbl.to_seq sr_rrds in
```

Comment:
This change is OK since their lifetime doesn't extend beyond the lifetime of the mutex